platform/i18n/src/locales/en-US/SegmentationTable.json

```diff
@@ -3,16 +3,24 @@
   "Add new segmentation": "Add new segmentation",
   "Add segment": "Add segment",
   "Add segmentation": "Add segmentation",
+  "Contour": "Contour",
+  "Contour tools": "Contour tools",
+  "Contour Segmentations": "Contour segmentations",
   "Delete": "Delete",
   "Display inactive segmentations": "Display inactive segmentations",
   "Export DICOM SEG": "Export DICOM SEG",
   "Download DICOM SEG": "Download DICOM SEG",
   "Download DICOM RTSTRUCT": "Download DICOM RTSTRUCT",
   "Fill": "Fill",
   "Inactive segmentations": "Inactive segmentations",
+  "Labelmap": "Label map",
```

Comment:
I think labelmap is used as one word, I might be wrong

Reply:
Interesting... DICOM uses [label map](https://dicom.nema.org/dicom/news/June2024/docs/sups/sup243-slides.pdf). However something like 3D Slicer refers to them as [labelmap](https://slicer.readthedocs.io/en/latest/user_guide/modules/segmentations.html). @sedghi I will let you choose 😉